Makefile

```diff
@@ -1,7 +1,7 @@
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 # Variable Definitions                                                                                            #
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-VERSION = $(shell git describe --match "v[0-9]*" --abbrev=0 --tags)
+VERSION = 2.25.1 #$(shell git describe --match "v[0-9]*" --abbrev=0 --tags)
```

Comment:
leftover from local testing?

Reply:
Yes, thanks @ryepup  for catching this. Removed that change.